### ISSUE TITLE
ruamel_yaml package not found

### ISSUE THREAD

**Issue by brynpickering** [calliope-project/calliope#71]
An issue that has occurred on both Windows and Linux devices since adding the new YAML parser, described in more detail in issue #69:

```
Hello, if the merge_loc_tech_sets branch is working on my computer, I am facing some troubles to install it on a Linux server.

I used checkout to be on the branch merge_loc_tech_sets and then I tried to install it :
python setup.py develop
which gave the following output:

['config/*.yaml', 'test/common/*.yaml', 'example_models/urban_scale/*.csv', 'example_models/urban_scale/*.yaml', 'example_models/urban_scale/*.rst', 'example_models/national_scale/*.csv', 'example_models/national_scale/*.yaml', 'example_models/national_scale/*.rst', 'example_models/urban_scale/model_config/*.csv', 'example_models/urban_scale/model_config/*.yaml', 'example_models/urban_scale/model_config/*.rst', 'example_models/national_scale/model_config/*.csv', 'example_models/national_scale/model_config/*.yaml', 'example_models/national_scale/model_config/*.rst', 'example_models/urban_scale/model_config/data/*.csv', 'example_models/urban_scale/model_config/data/*.yaml', 'example_models/urban_scale/model_config/data/*.rst', 'example_models/national_scale/model_config/data/*.csv', 'example_models/national_scale/model_config/data/*.yaml', 'example_models/national_scale/model_config/data/*.rst', 'test/common/t_time/*.csv', 'test/common/t_constraints_from_file/*.csv', 'test/common/t_1h/*.csv', 'test/common/t_positive_demand/*.csv', 'test/common/t_6h/*.csv', 'test/common/t_erroneous/*.csv']
running develop
running egg_info
writing calliope.egg-info/PKG-INFO
writing dependency_links to calliope.egg-info/dependency_links.txt
writing entry points to calliope.egg-info/entry_points.txt
writing requirements to calliope.egg-info/requires.txt
writing top-level names to calliope.egg-info/top_level.txt
reading manifest file 'calliope.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'calliope.egg-info/SOURCES.txt'
running build_ext
Creating /home/aleroy/miniconda3/lib/python3.6/site-packages/calliope.egg-link (link to .)
Removing calliope 0.5.3 from easy-install.pth file
Adding calliope 0.5.3 to easy-install.pth file
Installing calliope script to /home/aleroy/miniconda3/bin

Installed /home/aleroy/Calliope/calliope
Processing dependencies for calliope==0.5.3
Searching for ruamel_yaml>=0.11
Reading https://pypi.python.org/simple/ruamel_yaml/
No local packages or working download links found for ruamel_yaml>=0.11
error: Could not find suitable distribution for Requirement.parse('ruamel_yaml>=0.11')
Apparently it can't find ruamel, even if it is installed (I could import ruamel.yaml in python).
Did anyone have the same problem or an idea how to solve it?

Thanks,
Arnaud
```

**Comment by brynpickering** [comment]
@arnaud-leroy did you have issues with installing it when you were on the master branch? You should be able to just install on the master branch, then the working version of Calliope will be whichever one you checkout.

i.e. if you have a locally cloned version of calliope associated with a conda environment then when you run ```$ calliope run run.yaml``` the version of calliope used for that run will be whichever branch you currently have checked out.

Try creating a whole new environment associated with the calliope local clone (follow the [dev guide](https://calliope.readthedocs.io/en/stable/user/develop.html)) and then checkout `merge_tech_loc_sets`

**Comment by arnaud-leroy** [comment]
I couldn't make it work with the branches. On Linux the running time of the `merge_tech_loc_sets` branch was strictly the same as the master branch, but it was not the case on Windows.

So I created a  followed the dev guide and created a new environment. It is working now on the Linux server I am using. 

Thanks 👍

**Comment by sjpfenninger** [comment]
Great! This was fixed in 153d4d8, so it should be resolved when working off the master branch. This will soon be rolled into v0.5.4.
